src/simulation/openables/BronzeHAMChest.ts

```diff
@@ -0,0 +1,40 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+
+/* Gem loots */
+// TODO: check back for wiki drop table update from another one in the repo
+const GemTable = new LootTable()
```

Comment:
the tables you have for all of these look the same, just make one HAM chest class and condense each of them into that

Reply:
They still have different opening keys, and I can't condense that without changing itemID into a array. I'll let them be and there is a possibility that they got slightly different drop rates according to some searching.

Reply:
what matters is the table not what opens it

Reply:
I can make a HAM loot table that I import to all the different chests if that's what you mean

Reply:
i dont understand why this is an issue when youve already done it before and these dont even have different tables